ftd/list.ftd

```diff
@@ -13,14 +13,14 @@ To declare a new list variable, you can use the following syntax:
 -- ds.code: Declaring a `list`
 lang: ftd
 
-\-- <data-type> list <list-name>:
+\-- &lt;data-type&gt; list &lt;list-name&gt;:
 
-\-- <value>
-\-- <value>
-\-- <value>
+\-- &lt;value&gt;
```

Comment:
@Heulitig, @Arpita-Jaiswal User is not supposed to do this, this is our internal headache and should be taken care by us.